update_conda_lock/action.yml

```diff
@@ -0,0 +1,129 @@
+name: 'Conda Lock Updating Action'
+description: 'Action Pushes a Branch with Conda Lock Update'
+
+inputs:
+  commit_message:
+    description: 'Message of the bump commit'
+    required: true
+    default: '[BOT] Bump Conda Lock'
+  user_name:
+    description: 'Name of the user creating commit'
+    required: true
+    default: 'CONDA_LOCK_UPDATING_BOT'
+  user_email:
+    description: 'Email address of the user creating commit'
+    required: true
+    default: '<>'
+  branch_name:
+    description: "Name of the pushed branch; it will be suffixed with GH Action's Run ID"
+    required: true
+    default: 'update_lock'
+  environment_file:
+    description: 'Path of the `environment.yml` file'
+    required: true
+    default: 'environment.yml'
+  conda_lock_file:
+    description: 'Path of the Conda Lock file (needs to have txt/yml/yaml extension)'
+    required: true
+    default: 'conda_lock.yml'
+  pr_title:
+    description: 'Title of the Pull Request; it will be suffixed with UTC time and date'
+    required: true
+    default: '[BOT] Bump Conda Lock'
+  github_token:
+    description: 'GitHub Access Token; use Private Access Token to trigger workflows issuing the Pull Request'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+  - id: lock-update
+    shell: bash
+    run: |
+      echo "::group::Check git"
```

Comment:
Please, add this to a shell script and call it from here.

Moreover, it seems that most of it is not specific to the action, but general info about the environment. Therefore, I suggest to create two scripts.